extensions/wrapper/wrapper-common-mappers/src/main/java/de/sovity/edc/ext/wrapper/api/common/mappers/utils/UiAssetCreator.java

```diff
@@ -0,0 +1,29 @@
+package de.sovity.edc.ext.wrapper.api.common.mappers.utils;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@RequiredArgsConstructor
+@Schema(description = "Asset Creator Details")
+public class UiAssetCreator {
```

Comment:
I think we should remove the Ui-Prefix for these helper classes, as they are not part of the UI API Wrapper models